src/routes/index.ts

```diff
@@ -18,3 +27,25 @@ export function applicationRoutes(app: Koa<any, {}>) {
   app.use(trackingPlanRoutes);
   app.use(userTransformRoutes);
 }
+
+export function addSwaggerRoutes(app: Koa<any, {}>) {
+  // Ading swagger routes, check swagger docs in http://localhost:5050/docs
+  if (enableSwagger) {
+    const router = new Router();
+    if (existsSync('./dist/src/swagger.json')) {
```

Comment:
Let's add error handling around the file operations logic.

Reply:
Fixed

Reply:
`existsSync` is also a file operation.

Reply:
Fixed